pages/osx/xcrun.md

```diff
@@ -0,0 +1,23 @@
+# xcrun
+
+> Run or locate development tools and properties.
+
+- Get help:
+
+`xcrun --help`
+
+- Get version:
+
+`xcrun --version`
```

Comment:
I'm not convinced of the usefulness of these examples. Could we replace them with a basic example of how to use the command please?

Reply:
I added what @pixelcmtd suggested, but I didn't really replace the first two examples and kept them. But, I'm a first-time contributor to this project, so if those two examples are unnecessary, just let me know!

Reply:
The page is quite short and OSX doesn't always use the standard `-h` and `-v` flags, but they should at least be at the bottom.

Reply:
@marchersimon Is that necessary? The "Guidelines" section in [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#Guidelines) implies that the single-letter option isn't necessary, but if it is, I can add it.

Reply:
I was using the short options just symbolically, because I was too lazy to write out `--help` and `--version`.

Reply:
Either way, both the `--help` and `--version` options should be at the bottom of the page, if kept at all.